File: Lab3Scripts/LAB3PYTHON/usuario.py
class Usuario:
    def __init__(self, nombre, id):
        self._nombre = nombre
        self._id = id

    def get_id(self):
        return self._id

    def get_nombre(self):
        return self._nombre

    def mostrar_informacion(self):
        print(f"Nombre: {self._nombre}, ID: {self._id}")

File: Lab3Scripts/LAB3JAVA/Usuario.java
public class Usuario {
    private String nombre;
    private int id;

    public Usuario(String nombre, int id) {
        this.nombre = nombre;
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }

    public void mostrarInformacion() {
        System.out.println("Nombre: " + nombre + ", ID: " + id);
    }
}
